site-manage/site-manage-impl/impl/src/bundle/UserAuditService.properties

```diff
@@ -1,7 +1,5 @@
 #M will display as Last Name, First Name (user's eid)
 M = {0} ({1})
 S = Self
-#M will display as Last Name, First Name (user's eid)
```

Comment:
Can you revert these changes? You're changing the default locale.

Reply:
these were double keys